ddtrace/internal/debug.py

```diff
@@ -137,3 +138,91 @@ def collect(tracer):
         partial_flush_enabled=tracer._partial_flush_enabled,
         partial_flush_min_spans=tracer._partial_flush_min_spans,
     )
+
+
+def pretty_collect(tracer, color=True):
+    class bcolors:
+        HEADER = "\033[95m"
+        OKBLUE = "\033[94m"
+        OKCYAN = "\033[96m"
+        OKGREEN = "\033[92m"
+        WARNING = "\033[93m"
+        FAIL = "\033[91m"
+        ENDC = "\033[0m"
+        BOLD = "\033[1m"
+
+    info = collect(tracer)
+
+    info_pretty = """{0}{1}Tracer Configurations:{2}
```

Comment:
```suggestion
    info_pretty = """{blue}{bold}Tracer Configurations:{end}
```

We can use kwargs here to `format` to make this a little more readable